app/javascript/components/analytics/display/WorkflowTable.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import InfoTooltip from '../../util/InfoTooltip';
+
+class WorkflowTable extends React.Component {
+  render() {
+    return (
+      <React.Fragment>
+        <div className="analytics-table-header">
+          {this.props.title}
+          {this.props.tooltipKey && (
+            <span className="h6 text-left">
+              <InfoTooltip tooltipTextKey={this.props.tooltipKey} location="right"></InfoTooltip>
+            </span>
+          )}
+        </div>
+        <table className="analytics-table text-right">
+          <thead>
+            <tr className="header">
+              <th></th>
+              <th>Exposure</th>
+              <th>Isolation</th>
+              <th>Total</th>
+            </tr>
+          </thead>
+          <tbody>
+            {this.props.data.map((row, r_index) => (
```

Comment:
how do you feel about camelCase here?

Reply:
I modeled it off of other instances in the app, but also think this is easier to read and decipher what it is.  do you have a strong opinion for camel case?

Reply:
React uses camelCase, so it's definitely preferable. But I dont really care. `r_index` is probs more readable than `rIndex`

Reply:
could do `rowIndex`?

Reply:
but that's more letters

Reply:
I'll leave this one up to you